public/manifest/development.json

```diff
@@ -0,0 +1,34 @@
+{
+  "v1": [
+    {
+      "country_code": "IN",
+      "endpoint": "http://localhost:3000/api/",
+      "display_name": "India",
+      "isd_code": "91"
+    },
+    {
+      "country_code": "BD",
+      "endpoint": "http://localhost:3000/api/",
+      "display_name": "Bangladesh",
+      "isd_code": "880"
+    },
+    {
+      "country_code": "ET",
+      "endpoint": "http://localhost:3000/api/",
+      "display_name": "Ethiopia",
+      "isd_code": "251"
+    },
+    {
+      "country_code": "US",
+      "endpoint": "http://localhost:3000/api/",
+      "display_name": "United States",
+      "isd_code": "1"
+    },
+    {
+      "country_code": "UK",
+      "endpoint": "http://localhost:3000/api/",
+      "display_name": "United Kingdom",
+      "isd_code": "44"
+    }
+  ]
+}
```

Comment:
This file isn't necessary anymore. I think it got re-added through some merge-fu.